template/fpe-api/context.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Core contexts
+ */
+import {createHook} from 'fpe-implementation';
+import {
+  RtcContext,
+  MinUidContext,
+  MaxUidContext,
+  PropsContext,
+  LocalContext,
+} from '../agora-rn-uikit';
+import {default as DeviceContext} from '../src/components/DeviceContext';
+import {default as StorageContext} from '../src/components/StorageContext';
+
+export const useRtcContext = createHook(RtcContext);
+export const useMinUidContext = createHook(MinUidContext);
+export const useMaxUidContext = createHook(MaxUidContext);
+export const usePropsContext = createHook(PropsContext);
+export const useLocalContext = createHook(LocalContext);
+export const useDeviceContext = createHook(DeviceContext);
+export const useStorageContext = createHook(StorageContext);
+
+export {LocalUserContext} from '../agora-rn-uikit';
```

Comment:
why this Provider needs to be exported seperately but not the others
TODO: Remove

Reply:
Done